src/audio/google/Kconfig

```diff
@@ -81,4 +81,18 @@ config GOOGLE_RTC_AUDIO_PROCESSING_MOCK
 	  Mock Google real-time communication audio processing.
 	  It allows for compilation check and basic audio flow checking.
 
+config GOOGLE_RTC_AUDIO_PROCESSING_MIC_BITS
```

Comment:
if we are converting to float why is this needed? Can't we just pull in hw params and convert?